### ISSUE TITLE
Hide In Plain Sights

### ISSUE THREAD

**Issue by msowby** [urothis/nwn-module-DungeonEternalX#4]
Feat is too strong for only requiring one level of Shadow Dancer and practically broken with a HIPs+Mage team.

Suggestion: Making a higher level requirement for HIPs (10 level of SD for example) or consider removing the feat entirely, subsequently giving more build variety to "corner sneakers"

**Comment by hotelechoxrayindiasierra** [comment]
Imo a good fix for this would be to give all races keen senses. Would make detectors a lot more viable. If you want to make a spotter you pretty much have to go elf.

**Comment by urothis** [comment]
The current change is "Increased HiPS time from 6 seconds to 24 so that you can hide every 4 rounds instead of every other round."

We wanted to try as small a change as possible to fix this without making hips just completely useless.

If this doesn't help the issue ingame, let us know here.

**Comment by hotelechoxrayindiasierra** [comment]
I'm not a huge fan of the above change. While I do think hipsers are more problematic here than other servers, I feel like adding that much of a CD would more or less invalidate them. It's a direct nerf on the feat. Giving everyone the ability to see, potentially, would open up more build variation. People don't even bother building spotters because without keen senses it's pretty useless in pvp. Limiting the feat to elves greatly curtails the options you have for builds. With this said, the above will absolutely deter hipsers in pvp lol.

**Comment by milliorn** [comment]
Giving everyone an Elf only feat diminishes elfs, so how does that solve a problem and not create another?

> Giving everyone the ability to see, potentially, would open up more build variation. People don't even bother building spotters because without keen senses it's pretty useless in pvp. Limiting the feat to elves greatly curtails the options you have for builds.

**Comment by hotelechoxrayindiasierra** [comment]
People already make elves as clerics and dex tanks because they're generally the most optimal choice. Elves wont be diminished by any capacity, it will instead allow people to make halforc and dwarf tanks that can get enough detect to see sneakers and hipsers.

**Comment by CoryNWN** [comment]
I think the keen senses option would open up more build variety on the server. A cooldown on hips just makes SD less viable, but with the keen-senses option people would be more inclined to play well thought-out stealth or detect builds.

But then again, I don't really like stealthers and would't mind not having to fight them.

**Comment by TertiusL** [comment]
please for the love of all that is holy... there is  a mage hipser now.. please kill these things with fire

**Comment by hotelechoxrayindiasierra** [comment]
Pretty easy to do that. Mages with hide in plain sight tend to not be very useful, contradictory to what most people think...

**Comment by TertiusL** [comment]
people seem to misunderstand my position on hipsers a lot, and i feel ive been pretty clear. IDGAF how hard/easy it is to kill them. IDGAF how useful they are or not. I REALLY DONT. My issue is - they kill any and all forms of FUN in team pvp. See what I just said? the FUN part... they remove it. they make it annoying to pvp... and i hate doing things that annoy me, and if i have the power to do something about, i will... as in i simply wont pvp. 

This server got <20 active players. 70% at the very least play for the pvp.. now lets remove the pvp.. you got 30% of 20 people left that will play on your server... the odds are stacked against the server. 

I repeat- hipsers arent OP, they arent game breaking, they arent usefull...... and very simply, they take away what little is left on the server. They ruin fun.

**Comment by hotelechoxrayindiasierra** [comment]
You're right, we should do our best to cater to the player base to make sure everyone is having fun. However... your opinion that hipsers aren't fun is just that, an opinion. It's formulated based on your own interests. Some other players really enjoy playing hipsers. I'm pretty indifferent on the subject but would lean towards helping out those who hate hipsers in PVP for the sake of preserving the player base. 

You're likely not having fun when a hipser shows up because you don't have a means to fight them. You don't have TS on your builds or you don't build detect. Detect is kinda like your saves... if your saves suck then you're going to have a bad day. People who don't build any detect whatsoever on their builds are asking for a bad time. It's a part of the game. You can build around hipsers and not have to worry about them, you have the power.

**Comment by TertiusL** [comment]
Yes, I am aware its my opinion, hence i stated my opinion in the discussion area. 


But it still feels like you are not listening to what I am saying. 2 toons in my entire vault does not have tts, due to all the hipsers. I have the means to counter them.... but i see i have to re iterate- I do not enjoy pvping cs hipsers. My 'opinion' is that i will simply sit out when hipser show up. because I cba to fight vs that cancer. You are making a ton of assumptions about why i dont want to face them after I already told you why i hate them.. they ruin my fun. and i can guarantee you I am not the only one. 

I am building around them, i just dont want to

**Comment by hotelechoxrayindiasierra** [comment]
I am definitely seeing what you're saying. That's why I've endeavored to meet you half way. So far you haven't really offered up any solutions to the problem. You've stated that you hate hipsers and will sit out when they come to pvp. I'm guessing that you'd simply like the feat to be removed from the game or made useless. We're trying to come to a compromise, not completely ruin the feat or just leave it the way it currently is. If your solutions to problems is to avoid them, then maybe that's just what you should do. You have to be willing to give a little.

**Comment by urothis** [comment]
^ this, we have 2 conflicting play styles that need to somehow meet in the middle so more people can enjoy the server. That's our goal here.

**Comment by milliorn** [comment]
There is a nwscript or nwnx function (cant recall) that checks for line of sight (might be in the exposed code for us to use to make functions with).  Point is, most people hate that you can use hips like an on/off switch, with or without corners to hide?  Well, then what if you check if anyone is in line of sight of the hips build, and if so, either disable the feats use, or change it to just how hiding works without the feat.

**Comment by TertiusL** [comment]
@milliorn then it will essetially be the normal hide...

@ hex, again you are making unbased assumptions. I have said various ideas for a solution. I am not only complaining, i have given my 2 cents on a solution. I am willing to meet halfway. Me sitting out is for how Hips are atm. I am just not going to ruin my day.

**Comment by urothis** [comment]
Provide a solution, we will discuss what will be affected by that solution. 
Repeat until we find a solution that satisfies the variables.

Github is not the place for anything but technical discussion.
Be respectful.

This goes for everyone.

I'm in the party that hips is bs and need to be nurfed.
Nothing should be so op to just sprinkle on nearly any build and it be stupid. (I know it's not actually this bad but it's too much.) 

I feel hips should be harder to obtain, requiring builds to require investment to obtain something so server pvp changing. 

This weekend I'm working on a diff from vanilla so we can look at everything that's modified from vanilla nwn in a singular place.
